sigs.yaml

```diff
@@ -62,8 +62,8 @@
 
 - name: ADSA
   description: Association of Data Science and Analytics
-  chairs: Srujan Gupta
-  meetingTime: ''
+  chairs: Harshit Agarwal
+  meetingTime: 'Follow on facebook.com/adsauiuc for meeting times'
```

Comment:
Can you put something smaller here @hagarwa3(like TBD or the actual meeting time)?  I don't think this text would fit on display